esp/src/eclwatch/TimingTreeMapWidget.js

```diff
@@ -211,7 +211,9 @@ define([
                     this.stdDev = Math.sqrt(variance);
                     for (var i = 0; i < timers.length; ++i) {
                         var prefix = "other";
-                        if (timers[i].Name.indexOf("Graph graph") == 0) {
+                        timers[i].__hpcc_name = timers[i].Name;
+                        if (timers[i].Name.indexOf("Graph graph") === 0) {
```

Comment:
Within String.prototype.IndexOf I do not think you can check for multiple instances of a string. The first parameter is the search value and the second _optional_ is the index. Also, zero in this case is if it _does exist_ -1 if it _does not exist_